packages/back-end/src/controllers/experiments.ts

```diff
@@ -531,6 +534,21 @@ export async function postExperiment(
 
   if (requiresWebhook) {
     await queueWebhook(req.organization.id);
```

Comment:
There are a lot more places that should trigger a cache invalidation.  `queueWebhook` is called from 7 different places and invalidation should probably be the same.  In fact, you could just abstract this to a single function  that both fires webhooks and invalidates CDNs.